src/explore-education-statistics-frontend/.env

```diff
@@ -1,3 +1,3 @@
 CONTENT_API_BASE_URL=http://localhost:5010/api
 DATA_API_BASE_URL=http://localhost:5000/api
-FUNCTION_API_BASE_URL=http://localhost:7071/api
+FUNCTION_API_BASE_URL=http://localhost:7073/api
```

Comment:
Did you intend to make this change?

Reply:
Yeh, my `run` script starts the notifier on 7073 rather than 7071. Doesn't seem like there are any default ports for the functions though, so I guess there technically isn't a 'right' setting for this right now.

Reply:
That's fine by me, thanks. I just mentioned it because running functions from Rider defaults to 7071. People not using the script will rarely need to run all the functions together so haven't run into port conflicts and have probably left all 3 function projects using 7071 in their local settings.  I will make everyone aware of the function ports that they should be using.